cdn/sign_url.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def signed_url url:, key_name:, key:, expiration:
+  # [START signed_url]
```

Comment:
consider including the `def signed_url` declaration in the shown sample. 

we think people will copy paste signed_url method and use it. therefore, the method signature matters.